test/algorithms/test_min_eigen_optimizer.py

```diff
@@ -245,6 +252,28 @@ def test_samples(self):
         self.assertAlmostEqual(result.fval, result.samples[0].fval)
         self.assertEqual(result.status, result.samples[0].status)
 
+    def test_bit_ordering(self):
```

Comment:
Could you add test of bit ordering for Grover and statevector too?

Reply:
Sure. I added unittests for GroverOptimizer and statevector as well.